theano/gpuarray/dnn.py

```diff
@@ -1446,12 +1443,13 @@ def get_op_params(self):
     def infer_shape(self, node, shape):
         return [shape[0], shape[1], shape[1]]
 
-    def make_node(self, x, scale, bias):
+    def make_node(self, x, scale, bias, epsilon=1e-4):
         ctx_name = infer_context_name(x, scale, bias)
         x = as_gpuarray_variable(x, ctx_name)
         scale = as_gpuarray_variable(scale, ctx_name)
         bias = as_gpuarray_variable(bias, ctx_name)
-        epsilon = as_scalar(self.epsilon).astype('float64')
+        assert (epsilon >= 1e-5)
```

Comment:
You can't do it there. epsilon could be a symbolic variable. Move this assert to the C code and make it raise an error.